packages/vx-network/src/Nodes.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import cx from 'classnames';
+import { Group } from '@vx/group';
+
+import DefaultNode from './DefaultNode';
+import { NodeProvidedProps } from './types';
+
+export type NodeProps<Node> = {
+  /** Array of links to render. */
+  nodes?: Node[];
+  /** Component for rendering a single link. */
+  nodeComponent:
+    | string
+    | React.FunctionComponent<NodeProvidedProps<Node>>
+    | React.ComponentClass<NodeProvidedProps<Node>>;
+  /** Classname to add to each node parent g element. */
+  className?: string;
+  /** Returns the center x coordinate of a node. */
+  x?: (d: Node) => number;
+  /** Returns the center y coordinate of a node. */
+  y?: (d: Node) => number;
+};
+
+export default function Nodes<Node>({
+  nodes = [],
+  nodeComponent = DefaultNode,
+  className,
+  x = (d: any) => (d && d.x) || 0,
```

Comment:
`x = (d:any) => ...` => `(d?: {x: number}) => ...`
`y = (d:any) => ...` => `(d?: {y: number}) => ...`

Reply:
this is similar to the [`defaultLink` above](https://github.com/hshoff/vx/pull/515#discussion_r328753856), where I get the error `Type 'Node' is not assignable to type '{ y: number; }'` (even if we default the `Node` type to `{ x, y }`, it's not guaranteed)